### DISCUSSION TITLE
Encountered a problem exporting data using pySpark from Doris 2 Iceberg with type DATETIME

### DISCUSSION THREAD

**Discussion by madeirak** [apache/doris#36482] [A - General / Q&A]
DATETIME in Doris -> String in Spark -> String in Iceberg
![image](https://github.com/apache/doris/assets/23215922/8eaa750b-0217-4e6b-b8a0-d2da0f5cdd6b)
but I want timetamp in Iceberg, not string, what is the suggested approach?

**Comment by ixzc** [comment]
what‘s the meaning of you? do you want to export your data from doris to iceberg by using pyspark?
